### ISSUE TITLE
[Bug]: nYnab Import Produces "An unknown error occurred while importing. Please report this as a new issue on Github."

### ISSUE THREAD

**Issue by al3x1337** [actualbudget/actual#2886]
### Verified issue does not already exist?

- [X] I have searched and found no existing issue

### What happened?

Trying to import a nYnab budget file having used both the yNab Exporter and the API method across both a self-hosted instance on Unraid with HTTPS and Pikapods, 
I've checked Actual-Server's directory and unraid container logs for any insight but could not find detailed logs.

### What error did you receive?

this **An unknown error occurred while importing. Please report this as a new issue on Github.** error occurs across both budget files that I used. 

![image](https://github.com/actualbudget/actual-server/assets/44815197/1826150f-4088-4bd8-b2f8-7986d587c6cd)

### Where are you hosting Actual?

Unraid and Pikapods

### What browsers are you seeing the problem on?

Firefox, Firefox Private Browsing

**Comment by MatissJanis** [comment]
👋 Thanks for the report. Would you mind providing an example file that we could use to reproduce the issue? Without the ability to reproduce there isn't much we can do to assist.

**Comment by al3x1337** [comment]
> 👋 Thanks for the report. Would you mind providing an example file that we could use to reproduce the issue? Without the ability to reproduce there isn't much we can do to assist.

Thank you for the comment. What I did and was use the Console in my brower's Dev Tools to read out what the error was, after realizing that is how to access any kind of logging.

In my case I had a Category Group named "Income" that was creating a conflict. Resolved the import error.

**Comment by youngcw** [comment]
@al3x1337 Thanks for looking into it.  I think that issue got fixed, in recent release.  If you still have that old export you could try again and see if its still broken

**Comment by al3x1337** [comment]
@youngcw For some context, I self-hosted on Unraid and did not see an update for Actual Server. Still, I added :latest to the repo tag and forced an update to make sure I'm on the latest version v24.6.0. After that I used the [YNAB JSON website](https://json-exporter-for-ynab.netlify.app/#) to export my Budget json before encountering the Import error.

Coincidentally all 3 of my existing budgets had the Income category so it wasn't until I created a new budget that I realized something within my Budget is breaking the importer. Glad I came across a way to access the logs since that gave away the smoking gun as to what the issue was.
